plugins/backfill/transformerExporter.go

```diff
@@ -15,8 +14,6 @@ var Exporter exporter
 
 func (e exporter) Export() ([]event.TransformerInitializer, []storage.TransformerInitializer, []interface1.ContractTransformerInitializer) {
 	return []event.TransformerInitializer{},
-		[]storage.TransformerInitializer{
-			dog_v1_3_0.StorageTransformerInitializer,
```

Comment:
This backfill also bumped into the throttling issue, right? Let's make sure we're tracking all the outstanding backfill work somewhere, so we don't miss it.